test/box-luatest/gh-5602-environment-vars-cfg_test.lua

```diff
@@ -0,0 +1,198 @@
+local t = require('luatest')
+local fio = require('fio')
+
+local justrun = require('luatest.justrun')
+local treegen = require('luatest.treegen')
+
+local g = t.group()
+
+local tmpdir = fio.tempdir()
+local listen_sock1 = fio.pathjoin(tmpdir, 'listen1.sock')
+local listen_sock2 = fio.pathjoin(tmpdir, 'listen2.sock')
+
+-- Check that environment cfg values are set correctly.
+g.test_gh_5602_tarantool_env_variables_1 = function()
+    local dir = treegen.prepare_directory({}, {})
+    local script_name = 'test_gh_5602_tarantool_env_variables_1.lua'
+    local script_body = ([[
+local test = require('tap').test('gh-5602')
+local status, err = pcall(box.cfg, {background = false, vinyl_timeout = 70.1})
+
+test:plan(6)
+test:ok(status, 'box.cfg is successful')
+test:is(box.cfg['listen'], '%s', 'listen')
+test:is(box.cfg['readahead'], 10000, 'readahead')
+test:is(box.cfg['strip_core'], false, 'strip_core')
+test:is(box.cfg['log_format'], 'json', 'log_format is not set')
```

Comment:
Nit: the 'is not set' phrase seems incorrect.

Reply:
Updated:


```diff
--- a/test/box-luatest/gh-5602-environment-vars-cfg_test.lua
+++ b/test/box-luatest/gh-5602-environment-vars-cfg_test.lua
@@ -20,7 +20,7 @@ test:ok(status, 'box.cfg is successful')
 test:is(box.cfg['listen'], '%s', 'listen')
 test:is(box.cfg['readahead'], 10000, 'readahead')
 test:is(box.cfg['strip_core'], false, 'strip_core')
-test:is(box.cfg['log_format'], 'json', 'log_format is not set')
+test:is(box.cfg['log_format'], 'json', 'log_format is correct')
 test:is(box.cfg['log_nonblock'], false, 'log_nonblock')
 
 os.exit(test:check() and 0 or 1, true) ]]):format(listen_sock)
```